### PR TITLE
uniform return values in base_url

### PR DESCRIPTION
**Pull request by r0fls**
Address #1338.

### CODE REVIEW

**Comment by r0fls** [comment]
Is this not running the circle ci tests?

File: redash/utils/__init__.py [hunk 1]
```diff
@@ -155,6 +155,6 @@ def base_url(org):
     if settings.MULTI_ORG:
         return "https://{}/{}".format(settings.HOST, org.slug)
 
-    return settings.HOST
+    return "https://{}".format(settings.HOST)
```
Comment (arikfr) [inline redash/utils/__init__.py]: Unfortunately not everyone is using `https`... I think uniforming should be in the opposite direction, and make the MULTI_ORG branch of the if not use https by default.

**Comment by r0fls** [comment]
@arikfr updated

**Comment by arikfr** [comment]
You kind of missing the point: some will have it as `http` others as `https`. So either way this will break it for someone. Also anyone who has this set already will have the protocol set there already so this change will break it for them.

I don't see why it's important to change this line. Let's leave it as is...

Thanks.

**Pull request closed without merging**